Content.Server/Fluids/EntitySystems/SpillableSystem.cs

```diff
@@ -111,6 +120,27 @@ private void SpillOnLand(EntityUid uid, SpillableComponent component, ref LandEv
         SpillAt(drainedSolution, EntityManager.GetComponent<TransformComponent>(uid).Coordinates, "PuddleSmear");
     }
 
+    private void AddEmptyVerb(EntityUid uid, SinkComponent component, GetVerbsEvent<Verb> args)
+    {
+        if (!args.CanAccess || !args.CanInteract || args.Using == null)
```

Comment:
popup feedback

Reply:
I believe there doesn't seem to need a popup feedback here, since the AppEmptyVerb isn't a player action